osier/equations.py

```diff
@@ -206,6 +212,7 @@ def annual_emission(
 
     return emissions_total
 
+annual_co2 = partial(annual_emission, attribute='co2')  # for backwards compatibility
```

Comment:
This doesn't go in a function somewhere? Seems like a strange place for this.

Reply:
It's for backwards compatibility. A previous version of osier had a function called `annual_co2` so I recreated it in the simplest manner possible.